bot/utilities/DbConnection.ts

```diff
@@ -0,0 +1,83 @@
+import { ConnectionPool, connect as mssqlConnect, Response } from "mssql";
+import { Storage, ConnectionStatus } from "./Global";
+
+export default class DbConnection implements Storage {
+  pool: ConnectionPool;
+  connectionStatus: ConnectionStatus = ConnectionStatus.Awaiting;
+
+  async connect(config) {
+    try {
+      this.pool = await mssqlConnect(config);
+      this.connectionStatus = ConnectionStatus.Connected;
+      console.log("Database connected successfully");
+    } catch (err) {
+      this.connectionStatus = ConnectionStatus.Error;
+      console.error(`Error establishing database connection: ${err}.`);
+    }
+  }
+
+  async select(table: string, filter?: any): Response {
```

Comment:
Since this is just a demo project I'm not too worried about it but using string interpolation like `SELECT * FROM ${table} WHERE ${filterString};` can lead to being exposed to SQL injection. Also, forgot to mention TypeOrm manages the database connection so I wouldn't worry too much about changing it regardless.

Reply:
The `mssql` library automatically sanitizes template literals, though it's not super apparent to what degree.
https://www.npmjs.com/package/mssql#es6-tagged-template-literals

Reply:
Discussion in Slack - We're pulling this out anyways and replacing with TypeOrm so LGTM